src/components/oscal-utils/OSCALProfileResolver.js

```diff
@@ -145,11 +139,11 @@ export function OSCALResolveProfile(
       parentUrl,
       profile["back-matter"],
       inheritedProfilesAndCatalogs.inherited,
-      onSuccess,
+      () => {
```

Comment:
Is additional wrapping function needed? Can we not just call `onSuccess(inheritedProfilesAndCatalogs)` directly?

Reply:
If we just eliminate the wrapping functions, then we would need to manually re-render the page.

Reply:
Could you elaborate on why that's the case?

Reply:
Disregard, I see what's going on there.

Reply:
When we don't use a wrapper function, JavaScript attempts to evaluate `onSuccess(inheritedProfilesAndCatalogs)` to a value and no React state is modified inside `OSCALResolveProfileOrCatalogUrlControls`. But when we do use a wrapper function, JavaScript simply evaluates it to a function object, which can easily be called to perform the necessary actions and modify a React state.

Manually re-rendering the page works because we modify `inheritedProfilesAndCatalogs.inherited` inside the function, which will eventually provide `OSCALProfileCatalogInheritance` with the necessary data.